tests/test_openclhpp.cpp

```diff
@@ -3378,4 +3378,47 @@ void testCommandBufferInfoKHRCommandQueues()
 #endif
 }
 
+/****************************************************************************
+ * Tests for allocate and deallocate API (clSVMAlloc, clSVMFree, clEnqueueSVMMap)
+ ****************************************************************************/
+
+int *testMemory;
+void *clSVMAllocARM_stubForMemoryAllocation(cl_context context,cl_svm_mem_flags flags,
+                         size_t size, cl_uint alignment, int cmock_num_calls)
+{
+    TEST_ASSERT_EQUAL_HEX(size, 3 * sizeof(int));
```

Comment:
Can we check some of the other parameters to this function also?

For example, the test explicitly requested `cl::SVMTraitCoarse`, so we should be able to check the flags.  Can we test the alignment too?